apps/openchallenges/challenge-service/src/main/resources/db/migration/V1.0.1__insert_temp_data.sql

```diff
@@ -84,6 +84,20 @@ VALUES (
     'PapersWithCode',
     'logo/papers-with-code.jpg',
     'https://paperswithcode.com/'
+  ),
+  (
+    13,
+    'eterna',
+    'Eterna',
+    '',
```

Comment:
Missing logo

Reply:
I previously relied on the orgs' Twitter accounts to get the hi-res logos, but since Twitter is no longer publicly available.  I had meant to get the logos at a later time, and I must've forgot.  Sorry!

Reply:
No worries! Since this PR adds a significant amount of data, I'll add the missing logo and then merge the PR after reviewing the data in the app.

Reply:
I have added the object key to the Google spreadsheet